android/src/main/java/com/iterable/reactnative/RNIterableAPIModule.java

```diff
@@ -238,7 +239,7 @@ public void getAttributionInfo(Promise promise) {
         IterableLogger.printInfo();
         IterableAttributionInfo attributionInfo = IterableApi.getInstance().getAttributionInfo();
         if (attributionInfo == null) {
-            promise.resolve(null);
+            promise.reject("", "No attribution info found");
```

Comment:
Hmm, I didn't pay attention to this earlier, it seems similar to email and userId.. The value is `null`, so it shouldn't be an error.